prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -416,6 +436,28 @@ public async Task<BusinessLicenceDocument> GetLatestBusinessLicenceAsync(int sit
                 .FirstOrDefaultAsync();
         }
 
+        public async Task<SiteRegistrationNote> CreateSiteRegistrationNoteAsync(int siteId, string note, int adminId)
+        {
+            var SiteRegistrationNote = new SiteRegistrationNote
+            {
+                SiteId = siteId,
+                AdjudicatorId = adminId,
+                Note = note,
+                NoteDate = DateTimeOffset.Now
+            };
+
+            _context.SiteRegistrationNotes.Add(SiteRegistrationNote);
+
+            var created = await _context.SaveChangesAsync();
+            if (created < 1)
+            {
+                throw new InvalidOperationException("Could not create site registration note.");
+            }
+            // TODO: Business events for sites?
```

Comment:
Seems like a good idea to start collecting data on sites from the start.  I would ask Jason what he might want to see in metabase.